Frameworks/CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -53,26 +55,12 @@
 #include <Foundation/NSTimeZone.h>
 #include <Foundation/NSDate.h>
 
-struct tzhead {
```

Comment:
I asked the same question in the previous CR before we moved this CR to github - no one replied, so I assumed they agreed that the scope of changes was large enough that // WINOBJC comments were not the most useful.

---
In reply to: [72718150](https://github.com/Microsoft/WinObjC/pull/701#discussion_r72718150) [](ancestors = 72718150)